Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/install/linux.md

</details>

> > The shell configuration file needs to be restarted or re-sourced

Reviewer: ```suggestion
> Restart your terminal, or re-source the shell configuration file
```
